dsaps/cli.py

```diff
@@ -15,7 +20,23 @@
 @click.pass_context
 def main(ctx, url, email, password):
     ctx.obj = {}
-    print('Application start')
+    structlog.configure(processors=[
```

Comment:
I was able to remove this entire block and it logged fine?

Reply:
Are you getting `2019-11-26 13:39.16 Application start` or `{"event": "Application start", "logger": "__main__", "level": "info", "timestamp": "2019-11-26T18:40:30.819933Z"}`? I just commented it out again and I'm still having the same problem where it gives me unstructured log entries in the terminal but nothing to the file